x-pack/elastic-agent/_meta/config/elastic-agent.docker.yml.tmpl

```diff
@@ -43,6 +43,9 @@ inputs:
 
 #     # optional values
 #     #protocol: "https"
+#     #service_token: "${FLEET_SERVER_SERVICE_TOKEN}"
```

Comment:
I've noticed in the `outputs` section that we do not have the ability to look for an api key within a docker container. Is this something we should create an issue for?

Reply:
@ruflin

Reply:
Yes, we should really have that.

Reply:
https://github.com/elastic/beats/issues/29508 I created an issue to track it